scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/SimulationUtility.java

```diff
@@ -12,13 +12,23 @@
 public final class SimulationUtility {
 
   private static MissionModel<?> makeMissionModel(final MissionModelBuilder builder, final SerializedValue config) {
-    final var factory = new GeneratedMissionModelFactory();
+    final var factory = new gov.nasa.jpl.aerie.banananation.generated.GeneratedMissionModelFactory();
     final var model = factory.instantiate(config, builder);
     return builder.build(model, factory.getTaskSpecTypes());
   }
 
+  public static MissionModel<?>
+  getFooMissionModel() {
+    var conf = new gov.nasa.jpl.aerie.foomissionmodel.Configuration();
```

Comment:
For consistency within file:
```suggestion
    final var conf = new gov.nasa.jpl.aerie.foomissionmodel.Configuration();
```